packages/altair-app/src/app/modules/altair/config.ts

```diff
@@ -122,6 +139,7 @@ export class AltairConfig {
     postRequestScript: '',
     instanceStorageNamespace: 'altair_',
     settings: (undefined as unknown as AltairConfigOptions['initialSettings']),
+    initialSubscriptionsProvider: undefined as AltairConfigOptions['initialSubscriptionsProvider']
```

Comment:
Did this work without typing to `unknown` first? 🤔

Reply:
yes, it worked

Reply:
Awesome. Wonder why it didn't work for me earlier